requirements.txt

```diff
@@ -1,8 +1,9 @@
 numpy>=1.17
 scipy>=1.4
-qiskit-terra>=0.21.1
+qiskit-terra @ git+https://github.com/nkanazawa1989/qiskit-terra.git@04a656ea99cc7769e82e1347924c2ab839272859#egg=qiskit_terra
```

Comment:
Note to self: we will need to change this to pulling from main once 8005 is merged and once Terra releases we will have t change again.